tests/cases/fourslash/completionListAtIdentifierDefinitionLocations_Generics.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path='fourslash.ts' />
+
+////interface A</*genericName1*/
+

```

Comment:
don't put newlines in here unless they also have //// preceding them, otherwise fourslash considers each thing a new file (all within the same project context). It's usually an unobservable difference but it's best to just be safe.